config/06-swagger/00-config/00-swagger-configmap.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   name: swagger
   namespace: tekton-hub
 data:
-  API_URL: 'https://api.hub.tekton.dev'
+  API_URL: 'https://raw.githubusercontent.com/tektoncd/hub/main/api/v1/gen/http/openapi3.yaml'
```

Comment:
we should point to a specific commit instead of main branch .. there might be new APIs added and release not done, this will show them in the swagger

Reply:
I guess we can keep this as it is over here and at the time of deploying we can make the API_URL to point to the particular tag by setting the env
https://github.com/tektoncd/hub/blob/6c97d9e8d2fda05a60d294e2759978c87dc57594/swagger/image/start.sh#L15
https://github.com/tektoncd/hub/blob/6c97d9e8d2fda05a60d294e2759978c87dc57594/config/06-swagger/00-config/00-swagger-configmap.yaml#L8

Reply:
I think we can use the raw url from the latest release ??
For example https://raw.githubusercontent.com/tektoncd/hub/v1.3.0/api/v1/gen/http/openapi3.yaml

Reply:
`https://api.hub.tekton.dev` this isn't working at the moment so I would avoid keeping it  :D

Reply:
by default let it point to main branch .. we can update in configmap while deploying

Reply:
good to merge?

Reply:
> by default let it point to main branch .. we can update in configmap while deploying

This can even help in development where user can directly see what's there in main